src/gl-matrix/vec4.js

```diff
@@ -561,4 +561,29 @@ vec4.equals = function (a, b) {
             Math.abs(a3 - b3) <= glMatrix.EPSILON*Math.max(1.0, Math.abs(a3), Math.abs(b3)));
 };
 
+/**
+ * Returns the cross-product of three vectors in a 4-dimensional space
+ *
+ * @param {vec4} result the receiving vector
+ * @param {vec4} U the first vector
+ * @param {vec4} V the second vector
+ * @param {vec4} W the third vector
+ * @returns {vec4} result
+ */
+vec4.cross = function (result, U, V, W) {
```

Comment:
For consistency with the rest of the code base, please name this variable "out" and make the other three lower case. Also, again for consistency, please move this function declaration up between `vec4.transformQuat` and `vec4.forEach`.